jerry-core/api/jerry.c

```diff
@@ -52,7 +52,18 @@
 
 JERRY_STATIC_ASSERT (sizeof (jerry_value_t) == sizeof (ecma_value_t),
                      size_of_jerry_value_t_must_be_equal_to_size_of_ecma_value_t);
-
+#if JERRY_BUILTIN_AGGREGATE_ERROR
+JERRY_STATIC_ASSERT ((int) ECMA_ERROR_NONE == (int) JERRY_ERROR_NONE
+                     && (int) ECMA_ERROR_COMMON == (int) JERRY_ERROR_COMMON
+                     && (int) ECMA_ERROR_EVAL == (int) JERRY_ERROR_EVAL
+                     && (int) ECMA_ERROR_RANGE == (int) JERRY_ERROR_RANGE
+                     && (int) ECMA_ERROR_REFERENCE == (int) JERRY_ERROR_REFERENCE
+                     && (int) ECMA_ERROR_SYNTAX == (int) JERRY_ERROR_SYNTAX
+                     && (int) ECMA_ERROR_TYPE == (int) JERRY_ERROR_TYPE
+                     && (int) ECMA_ERROR_URI == (int) JERRY_ERROR_URI
+                     && (int) ECMA_ERROR_AGGREGATE == (int) JERRY_ERROR_AGGREGATE,
+                     ecma_standard_error_t_must_be_equal_to_jerry_error_t);
```

Comment:
Keep the original assert, and add another which checks ECMA_ERROR_AGGREGATE only.